cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -252,6 +253,183 @@ describe('Dashboards', () => {
+      )
+      cy.getByTestID('variable-dropdown--Power').should('be.visible')
+      cy.getByTestID('variable-dropdown-input-typeAhead--Power').should(
+        'have.value',
+        'base'
+      )
+
+      makeGraphSnapshot().shouldBeSameAs(viewGraphOrig)
+
+      cy.getByTestID('cell--view-empty markdown').should(
+        'contain.text',
+        'The cat went here and there'
+      )
+
+      // Prep verification that changes in clone are not reflected in original,
+      // i.e. that they are fully decoupled
+      // change variable N.B. final assert is waiting on fix for #3287 see below
+      cy.getByTestID('variable-dropdown-input-typeAhead--Power').click()
+      cy.getByTestID('variable-dropdown--item')
+        .eq(4)
+        .click()
+      // const viewGraphCopy = makeGraphSnapshot()
+      cy.getByTestID('cell Name this Cell').within(() => {
+        cy.getByTestID('cell-context--toggle').click()
+      })
+
+      const replaceText =
+        '### William Blake\n#### Augeries of Innocense\n\n To see a World in a Grain of Sand\n\nAnd a Heaven in a Wild Flower\n\n...`'
+      cy.getByTestID('cell-context--note').click()
+      cy.getByTestID('markdown-editor')
+        .click()
+        .type('{ctrl}a')
+        .type(replaceText)
+
+      cy.getByTestID('save-note--button').click()
+
+      cy.getByTestID('nav-item-dashboards').click()
+
+      // Verify labels
+      cy.getByTestID('dashboard-card')
+        .eq(1)
+        .within(() => {
+          cy.getByTestID(`label--pill ${labelName}`).should('be.visible')
+          cy.getByTestID('dashboard-card--name').should(
+            'contain.text',
+            `${localDashName} (clone 1)`
+          )
+          cy.getByTestID('inline-labels--add').click()
+        })
+
+      cy.getByTestID(`label--pill ${secondLabelName}`).click()
+
+      cy.getByTestID('dashboard-card')
+        .eq(1)
+        .within(() => {
+          cy.getByTestID(`label--pill ${secondLabelName}`).should('be.visible')
+          cy.getByTestID(`label--pill--delete ${labelName}`).click()
+          cy.getByTestID(`label--pill ${labelName}`).should('not.exist')
+        })
+
+      cy.getByTestID('dashboard-card')
+        .first()
+        .within(() => {
+          cy.getByTestID(`label--pill ${secondLabelName}`).should('not.exist')
+          cy.getByTestID(`label--pill ${labelName}`).should('exist')
+        })
+
+      // Verify original is decoupled from clone
+      cy.getByTestID('dashboard-card')
+        .first()
+        .within(() => {
+          cy.getByTestID('dashboard-card--name').click()
+        })
+
+      /*
+      // TODO verify graph and variable in original once #3287 is fixed
+
+      cy.getByTestID('variable-dropdown-input-typeAhead--Power').should('have.value','base')
+      makeGraphSnapshot().shouldBeSameAs(viewGraphCopy, false)
+
+       */
+      cy.getByTestID('cell--view-empty markdown').should(
+        'contain.text',
+        'The cat went here and there'
+      )
```

Comment:
same here using

```ts
cy.contains('The cat went here and there')
```

Reply:
replaced with contains

Reply:
On second though decided to restore the assertion `.should()`.  See reasons in previous comment.  At his point we just want to assert that the note cell is unchanged, that it contains the original text and that the note is in no way coupled with any changes in the clone.